src/platform/site-wide/sass/modules/_m-side-nav.scss

```diff
@@ -1,5 +1,3 @@
-$side-nav-color-gray-lightest: #f8f8f8; // I could not find the color in our design utilities.
```

Comment:
![image](https://user-images.githubusercontent.com/2008881/119393671-17733d80-bc86-11eb-9ce9-afb5b9681c6f.png)
![image](https://user-images.githubusercontent.com/2008881/119393727-2eb22b00-bc86-11eb-9cd7-c535708656ce.png)


This is pretty close to `$color-gray-lightest`